packages/expo-image/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🎉 New features
 
+- [Android] Add tracing.
```

Comment:
nit: maybe put the changelog to the "Others` section, because from user's perspective they may not understand what the new feature "add tracing" really means.